.changeset/long-planets-sleep.md

```diff
@@ -0,0 +1,5 @@
+---
+'tabbable': patch
+---
+
+Fixed bug in `isDisabledFromFieldset`. The function wasn't checking whether the disabled <fieldset> containing `node` is the top-most disabled <fieldset>.
```

Comment:
Please put `<fieldset>` into backticks so it's rendered as code. Otherwise, Markdown accepts HTML and an open `<fieldset>` tag in there will mess-up the CHANGELOG.

Also, I like to reference the issue the fix is related to at the end, like this: `([#596](https://github.com/focus-trap/tabbable/issues/596))` (you have to link it because this is going into the CHANGELOG, which isn't a PR or an Issue, so just `#596` won't result in an auto-link from GitHub).